libs/blocks/hero-marquee/hero-marquee.css

```diff
@@ -0,0 +1,581 @@
+:root {
+  --s-min-height: 248px; /* 360px */
+  --m-min-height: 448px; /* 560px */
+  --l-min-height: 588px; /* 700px */
+  
+  /* 112 = 56px default padding */
+}
+
+.hero-marquee {
+  padding: var(--spacing-xl) 0;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacing-m);
+  width: 100%;
+  overflow: hidden;
+  min-height: var(--m-min-height);
+  justify-content: center;
+  align-items: center;
+}
+
+.hero-marquee.s-min-height { min-height: var(--s-min-height);}
+.hero-marquee.l-min-height { min-height: var(--l-min-height);}
+
+.dark .hero-marquee,
+.hero-marquee.dark {
+  color: var(--color-white);
+}
+
+.hero-marquee .background picture {
+  line-height: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+}
+
+/* Alignment */
+.hero-marquee .action-area,
+.hero-marquee .lockup-area {
+  display: flex;
+}
+
+.hero-marquee .action-area {
+  flex-flow: column wrap;
+  gap: var(--spacing-s); 
+  width: 100%;
+}
+
+.hero-marquee .background-split picture img {
+  object-fit: cover;
+  height: 100%;
+  width: 100%;
+}
+
+.hero-marquee.fg-media-cover picture {
+  line-height: 0em;
+  display: block;
+}
+
+.hero-marquee .main-copy {
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacing-xs);
+  justify-content: center;
+}
+
+/* Lockup Area */
+.hero-marquee .lockup-area { 
+  font-weight: 700;
+  display: flex;
+  align-items: center;
+  gap: var(--spacing-xs);
+  margin: 0 0 var(--spacing-xxs); 
+  font-size: var(--type-body-m-size);
+  line-height: var(--type-body-m-lh);
+  text-transform: initial;
+  white-space: nowrap
+} 
+
+.hero-marquee .lockup-area picture { 
+  line-height: 0;
+  display: block;
+}
+
+.hero-marquee .lockup-area picture img {
+  width: auto;
```

Comment:
Is the image limited to its container somewhere else?

Reply:
No, this selector does nothing and has been removed.